LayoutTests/imported/w3c/web-platform-tests/digital-credentials/allow-attribute.https-expected.txt

```diff
@@ -1,9 +1,9 @@
 
 
 PASS Policy to use: null, is cross-origin: false, is allowed by policy: true
-FAIL Policy to use: null, is cross-origin: true, is allowed by policy: false assert_equals: <iframe src="https://127.0.0.1:9443/digital-credentials/support/iframe.html" data-expect-is-allowed="false"></iframe> - Not implemented expected "NotAllowedError" but got "NotSupportedError"
```

Comment:
typically don't expect to see assertions showing in the test results !

Reply:
yeah, these are hard to debug without them. Why we left them in there.